tests/src/test/java/alluxio/client/cli/fsadmin/metrics/MetricsCommandIntegrationTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.cli.fsadmin.metrics;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
+import alluxio.client.cli.fsadmin.AbstractFsAdminShellTest;
+
+import alluxio.client.file.FileSystemTestUtils;
+import alluxio.grpc.WritePType;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.Test;
+
+import java.io.IOException;
+
+/**
+ * Integration tests for the metrics command.
+ */
+public final class MetricsCommandIntegrationTest extends AbstractFsAdminShellTest {
+  @Test
+  public void clearLeadingMasterMetrics() throws IOException {
```

Comment:
can there be a test for the worker clear as well?

Reply:
Yeah, but there wasn't a good way to get and check the worker metrics in the integration tests.
The web UI is not enabled in local alluxio cluster in integration tests, so `<host>:<web_port>/metrics/json` cannot be accessed. The `fsadmin report metrics` doesn't show any metrics related to the worker metrics (doesn't show any aggregated worker metrics) because the current `MetaMasterClient.getMetrics()` only expose a limited set of master metrics.

I plan to merge this PR first, in the next PR, I will modify the current `MetaMasterClient.getMetrics`, modify this method, move it to metricsMaster.
After that, the `fsadmin report metrics` command will also be modified to expose aggregated cluster metrics.